iothub/device/src/Transport/AmqpIot/AmqpIotExceptionAdapter.cs

```diff
@@ -40,9 +40,7 @@ internal static Exception ConvertToIotHubException(Exception exception)
         internal static Exception ConvertToIotHubException(Exception exception, AmqpObject source)
         {
             Exception ex = ConvertToIotHubException(exception);
-            if (source.IsClosing() &&
-                (ex is InvalidOperationException
-                || ex is OperationCanceledException))
+            if (source.IsClosing() && ex is InvalidOperationException)
```

Comment:
missed this from a previous PR of unwrapping OperationCanceledException from other exceptions in the hub device client